code/parse/sexp.cpp

```diff
@@ -4261,6 +4296,30 @@ void localize_sexp(int text_node, int id_node)
 	lcl_ext_localize(xstr.c_str(), Sexp_nodes[text_node].text, TOKEN_LENGTH - 1);
 }
 
+// Advance to and consume the closing parenthesis of a sexp, in case of a parse error.
+// If a closing parenthesis is not found, this advances to the end of the string.
+void skip_sexp()
+{
+	int level = 1;
+
+	while (*Mp != '\0')
+	{
+		if (*Mp == ')')
+		{
+			level--;
+			if (level == 0)
+			{
+				Mp++;
+				return;
+			}
+		}
+		else if (*Mp == '(')
```

Comment:
Open parentheses within quoted strings should be fine.  It's only in the middle of variables, containers, operators, or numbers that they would cause problems.

Reply:
Oh wait, I see what you mean.